server/server_test.go

```diff
@@ -304,3 +304,32 @@ func verifyGetResponse(t *testing.T, r *http.Response, expectedBytes []byte) {
 	require.NotEqual(t, "", r.Header.Get("Last-Modified"))
 	require.Equal(t, "", r.Header.Get("Pragma"))
 }
+
+// RotateKey supports only timestamp and snapshot key rotation
+func TestRotateKeyEndpoint(t *testing.T) {
+	ctx := context.WithValue(
+		context.Background(), "metaStore", storage.NewMemStorage())
+	ctx = context.WithValue(ctx, "keyAlgorithm", data.ED25519Key)
+
+	ccc := utils.NewCacheControlConfig(10, false)
+	handler := RootHandler(nil, ctx, signed.NewEd25519(), ccc, ccc, nil)
+	ts := httptest.NewServer(handler)
+	defer ts.Close()
+
+	rolesToStatus := map[string]int{
+		data.CanonicalTimestampRole: http.StatusOK, // just returning same as GetKey endpoint right now
```

Comment:
Non-blocking - I'm not sure I understand the comment about the `GetKey` endpoint?